### ISSUE TITLE
dalamud err: could not get swapchain address

### ISSUE THREAD

**Issue by MaxineMutini** [goatcorp/Dalamud#48]
game crashes on startup. ive dived through the logs and found that the game exits directly after `[INF] Started dalamud!`. so i dug around and found the dalamud log files and discovered the following error:
```
[ERR] Could not get SwapChain address via sig method, falling back to vtable...
System.Collections.Generic.KeyNotFoundException: Can't find a signature of 48 8B C4 55 41 54 41 55 41 56 41 57 48 8D 68 ?? 48 81 EC C0 00 00 00
   at Dalamud.Game.SigScanner.Scan(IntPtr baseAddress, Int32 size, String signature)
   at Dalamud.Game.SigScanner.ScanModule(String signature)
   at Dalamud.Game.Internal.DXGI.SwapChainSigResolver.Setup64Bit(SigScanner sig)
   at Dalamud.Game.Internal.BaseAddressResolver.Setup(SigScanner scanner)
   at Dalamud.Interface.InterfaceManager..ctor(Dalamud dalamud, SigScanner scanner)
```
this error was caused by simply logging in. the game window opens up, gets to the "Square Enix" logo, then immediately crashes.

  Operating System: Win10x64 v1909
  XIVLauncher version: xivlauncher-4.7.3.0-a55b6fb
  Launching as admin?: no, doing so gives different errors. if you want those, ill make a new report
  Using Steam?: no
  Addons/Plugins enabled: Rich Presence

**Comment by mattavist** [comment]
Yes that was the problem, had to check "Unblock" for all applications and dll files in Hooks. FFXIV crashed after loading, dalamud log attached.

[dalamud.txt](https://github.com/goaaats/Dalamud/files/4303458/dalamud.txt)

**Comment by ff-meli** [comment]
Thanks, hat should help with one problem, but maybe not with the crashes...

I made another build [here](https://github.com/ff-meli/Dalamud/releases/download/test2/Hooks.zip) that has some different logging and a fix for part of the signature error (maybe).  Could you give that one a try when you get a chance?  Same process as before, and you'll probably have to unblock things again.

Thanks!

**Comment by mattavist** [comment]
Another crash, another log!

[dalamud.txt](https://github.com/goaaats/Dalamud/files/4304365/dalamud.txt)

**Comment by ff-meli** [comment]
Thanks!  This helps narrow things down a bit, at least.

Test builds are going to be on hold a bit because of the patch, which I assume will derail dalamud in general for a day or two.

I'll make another after that, though it will probably just be adding logging to absolutely everything in the path that crashes, since it won't be nice and give us actual crash info (at least I know whereabout it is now though!)

**Comment by ff-meli** [comment]
Sorry for the delay on this!  Client patches and such are always a bit of a headache.

If you're still able to test, I have a new version [here](https://github.com/ff-meli/Dalamud/releases/download/test3/Hooks.zip)

This should still crash, but it logs just about everything possible, so it should hopefully pinpoint what the problem is, and then I can start trying to figure out why.

Steps to test are the same as before -
* Go to %APPDATA%\XivLauncher\addon\
* Delete the entire Hooks folder
* Unzip the test archive you downloaded above, directly into the "addon" folder. It should create a replacement Hooks folder with everything necessary inside. (Make sure you don't end up with Hooks\Hooks)
* Go through all the .exe files in the new Hooks folder and unblock them
* Run XivLauncher, make sure in-game features are enabled, launch the game
* Probably crash
* Upload the %APPDATA%\XivLauncher\addon\Hooks\dalamud.txt file here.

Thanks!

**Comment by ff-meli** [comment]
I won't be keeping the test builds up to date, since it's a bit of a chore to do.

If anyone is still having this issue and willing to test, comment here and I can see about making a new one.

**Comment by daemitus** [comment]
Please re-open a new issue if this issue persists.
